### PR TITLE
UI Readme Change

### PR DESCRIPTION
**Pull request by Amstutz**
This is a slight change in the Readme in order to make UI Components PR's easier to read for people with little experience with GIT.

### CODE REVIEW

File: src/UI/README.md [hunk 1]
```diff
@@ -203,24 +203,38 @@ If you would like to implement a new component to the framework you should perfo
     public function demo($demo){}
     ```
 
-6. Congratulations, at this point you are ready to present your work to the JF.
- However, it would be wise to enhence your work with a little mockup. This makes
- it much easier to discuss the new component at the JF. So best create such an example
- located e.g. at src/UI/examples/Demo/html.php:
+6. Congratulations, at this point you are ready to present your work to the JF. Create
+ a PR named "UI NameOfTheComponent". To make it easy for non-developers to
+ follow the discussion, you MUST link to the changed/added factory classes and mock in the
```
Comment (klees) [inline src/UI/README.md:208]: We should reflect this new MUST in rules.md, IMO.

Comment (Amstutz) [reply src/UI/README.md:208]: Done.

**Comment by atoedt** [comment]
Thank you so much for considering my needs.

**Comment by matthiaskunkel** [comment]
Jour Fixe: PR is highly appreciated. Can be merged to trunk.

**Pull request merged**